hibernate-core/src/main/java/org/hibernate/type/descriptor/java/PrimitiveByteArrayTypeDescriptor.java

```diff
@@ -73,7 +73,7 @@ public String extractLoggableRepresentation(byte[] value) {
 		byte[] bytes = new byte[string.length() / 2];
 		for ( int i = 0; i < bytes.length; i++ ) {
 			final String hexStr = string.substring( i * 2, (i + 1) * 2 );
-			bytes[i] = (byte) (Integer.parseInt(hexStr, 16) + Byte.MIN_VALUE);
+			bytes[i] = (byte) Integer.parseInt( hexStr, 16 );
```

Comment:
This old code wasn't consistent with how JDBC drivers handle the conversion from unsigned to signed bytes.